RELEASE_GUIDE.md

```diff
@@ -0,0 +1,115 @@
+# SOP: OpenSearch-Benchmark Release Guide
+
+- [Overview](#overview)
+- [Branches](#branches)
+    - [Releasing Branches](#release-branches)
+    - [Feature Branches](#feature-branches)
+- [Release Labels](#release-labels)
+- [Prerequisites](#prerequisites)
+- [Release new version of OpenSearch Benchmark to PyPi, Docker, and ECR](#release-new-version-of-opensearch-benchmark-to-pypi-docker-and-ecr)
+- [Error Handling](#error-handling)
+
+## Overview
+This document explains the release strategy for artifacts in this organization.
+
+## Branches
+
+### Release Branches
+
+Given the current major release of 1.0, projects in this organization maintain the following active branches.
+
+* **main**: The next `1.x` release. This is the branch where all merges take place and code moves fast.
+* **1.0**: The _current_ release. In between minor releases, only hotfixes (e.g. security) are backported to `1.0`.
+
+Label PRs with the next major version label (e.g. `2.0.0`) and merge changes into `main`. Label PRs that you believe need to be backported as `1.x` and `1.0`. Backport PRs by checking out the versioned branch, cherry-pick changes and open a PR against each target backport branch.
+
+### Feature Branches
+
+Do not creating branches in the upstream repo, use your fork, for the exception of long lasting feature branches that require active collaboration from multiple developers. Name feature branches `feature/<thing>`. Once the work is merged to `main`, please make sure to delete the feature branch.
+
+## Release Labels
+
+Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
+
+The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [maintainers](MAINTAINERS.md).
+
+## Prerequisites
+
+* Since releases are only done on Thursdays, maintainers should ensure all changes are merged by Tuesday.
+    * A week prior to the scheduled release, maintainers should announce a code freeze in the [#performance channel](https://opensearch.slack.com/archives/C0516H8EJ7R) within the OpenSearch Slack community. See the following example for what that might look like:
+```
+OpenSearch Benchmark release is scheduled for 1/25 and a code freeze will be put in place starting on 1/23.
+```
+* Ensure that version.txt matches the new release version before proceeding. If not, open a PR that updates the version in version.txt and merge it in before proceeding with the following steps. For example, if OSB is currently on version 0.3.0 and we want to release the next version as 0.4.0, update version.txt from 0.3.0 to 0.4.0.
+* Ensure you have git cloned the official OpenSearch Benchmark repository with the ssh address on your local computer.
```

Comment:
you have cloned the official OpenSearch Benchmark git repository.

Does it matter if it is the local computer?  This could be done on an EC2 instance.